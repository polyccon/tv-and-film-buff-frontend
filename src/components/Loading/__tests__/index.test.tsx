import React from 'react';
import { create } from "react-test-renderer";

import Loading from "../";

test("<Loading />", () => {
  const loading = create(<Loading />);
  expect(loading.toJSON()).toMatchSnapshot();
});
