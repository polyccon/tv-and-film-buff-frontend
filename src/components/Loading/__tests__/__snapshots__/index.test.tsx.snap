// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> 1`] = `
<div
  aria-busy={true}
  aria-live="polite"
  className="ant-spin ant-spin-spinning"
>
  <span
    className="ant-spin-dot ant-spin-dot-spin"
  >
    <i
      className="ant-spin-dot-item"
    />
    <i
      className="ant-spin-dot-item"
    />
    <i
      className="ant-spin-dot-item"
    />
    <i
      className="ant-spin-dot-item"
    />
  </span>
</div>
`;
